de.flose.Kochbuch.json

```diff
@@ -0,0 +1,98 @@
+{
+    "id": "de.flose.Kochbuch",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.mono6"
+    ],
+    "command": "kochbuch",
+    "finish-args": [
+        "--filesystem=home",
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11"
+    ],
+    "modules": [
+        {
+            "name": "mono",
+            "buildsystem": "simple",
+            "build-commands": [
+                "/usr/lib/sdk/mono6/install.sh",
+                "for lib in System.Drawing System.Windows.Forms System.Data System.Transactions Mono.Data.Sqlite Mono.Security System.Numerics; do cp -ar /usr/lib/sdk/mono6/lib/mono/gac/${lib} /app/lib/mono/gac/ && rm -f /app/lib/mono/gac/${lib}/*/*.pdb; done",
+                "cp /usr/lib/sdk/mono6/lib/mono/4.5/cert-sync.exe /app/lib/mono/4.5/"
+            ],
+            "build-options": {
+                "no-debuginfo": true,
+                "strip": true
+            }
+        },
+        {
+            "name": "libgdiplus",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.mono-project.com/sources/libgdiplus/libgdiplus-6.1.tar.gz",
+                    "sha256": "97d5a83d6d6d8f96c27fb7626f4ae11d3b38bc88a1726b4466aeb91451f3255b"
+                }
+            ]
+        },
+        {
+            "name": "kochbuch",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -dm755 /app/opt",
```

Comment:
`/app/opt` really isn't a standard path. Should just be all in `/app`

Reply:
I've removed the `opt` folder.